autoload/lsp/ui/vim/completion.vim

```diff
@@ -134,13 +134,14 @@ function! s:resolve_completion_item(completion_item, server_name) abort
   let l:ctx = {}
   let l:ctx['response'] = {}
   function! l:ctx['callback'](data) abort
-    let l:self['response'] = a:data.response
+    let l:self['response'] = a:data['response']
   endfunction
 
   call lsp#send_request(a:server_name, {
```

Comment:
should we have a try..catch here?

Reply:
Oh! Thanks for your review.
Yes, we should add try-catch here.